src/app/features/email-template-selection-dialog/email-template-selection-dialog.component.html

```diff
@@ -0,0 +1,50 @@
+<mat-dialog-content>
+  <h2 i18n>Select Email Template</h2>
+  <p>
+    <span i18n>
+      Select the template to use for sending an email to this record. If you
+      want to create a new template, you can do so from -
+    </span>
+    <a
+      [routerLink]="EmailTemplate.route"
+      *appDisabledEntityOperation="{
+        entity: EmailTemplate,
+        operation: 'update',
+      }"
+      matDialogClose
+      i18n
+      >configure available Email templates</a
+    >
+  </p>
+  <p>
+    <span i18n>
+      Note: Nothing will happen unless you have a default email client set up.
+      Please configure one to proceed.
+    </span>
+  </p>
+
+  <app-entity-select
+    [entityType]="EmailTemplate.ENTITY_TYPE"
+    [multi]="false"
+    [additionalFilter]="filteredTemplate"
+    label="Email Template"
+    i18n-label
+    placeholder="Select a email template"
+    i18n-placeholder
+    [form]="emailTemplateSelectionForm"
+    [disableCreateNew]="true"
```

Comment:
hmm, we could even allow this here? I hadn't thought about the inline "create new" function in this context yet.